pyproject.toml

```diff
@@ -23,10 +23,10 @@ nautobot = "^1.4.0"
 
 #
 # Optional dependencies installed as extras
-# 
+#
 
 # AWS SDK for Python (always keep in sync w/ dev dependency)
-boto3 = { version = "~1.19.5", optional = true }
+boto3 = { version = ">=1.19.0", optional = true }
```

Comment:
Generally we don't want fully-unbound dependencies as there's always the possibility that a new release will contain breaking changes. Looking at the changelog for boto3 there aren't any *documented* breaking changes in any 1.x.y release, so perhaps `>=1.19.0,<2.0` would be acceptable here?